WooCommerce/Classes/AppDelegate.swift

```diff
@@ -102,6 +103,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         setupUniversalLinkRouter()
         disableAnimationsIfNeeded()
 
+        configureWordPressKit()
```

Comment:
I like how this is a function and not an inline call of the flag check + conditional configuration logic 😄